src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -296,6 +297,10 @@ private async Task PreRunConditions(WorkerRuntime workerRuntime)
                     ColoredConsole.WriteLine("Could not find a valid .csproj file. Skipping the build.");
                 }
             }
+            else if (workerRuntime == WorkerRuntime.powershell && !CommandChecker.CommandExists("dotnet"))
+            {
+                throw new CliException("dotnet sdk is required for PowerShell functions. Please install https://microsoft.com/net");
```

Comment:
It would be good to specify .NET Core SDK is required (if so), because we have gotten questions on whether they need dotnet framework or core. And might be a good idea to use the same link as used below, just to keep things consistent :)
Or if you like this link better, feel free to update the constants below.
https://github.com/Azure/azure-functions-core-tools/blob/3815dbd2554f7658b0a1b8dd38c0498a1f346d2e/src/Azure.Functions.Cli/Common/Constants.cs#L56

Reply:
I agree. We should keep consistency. Exception message and link updated.